src/coreclr/tools/aot/ILCompiler.ReadyToRun/Interop/IL/Marshaller.ReadyToRun.cs

```diff
@@ -130,6 +130,18 @@ public static bool IsMarshallingRequired(MethodDesc targetMethod)
             return false;
         }
 
+        public static bool IsMarshallingRequired(MethodSignature methodSig, ModuleDesc moduleContext, UnmanagedCallingConventions callingConvention)
```

Comment:
The `callingConvention` parameter is not used, is it intentional?

Why do we need a new overload though? I would expect the existing `IsMarshallingRequired` to get an extra `callingConvention` argument if answer to this now depends on the calling convention as well (or why can we still decide it without the calling convention in the other overload?).

Reply:
In the other overload, we can extract the calling convention from the MethodSignature. This overload is for UnmanagedCallersOnly methods where the callconv is in the attribute, not the managed signature.

Reply:
The parameter is still unused, is that intentional?

Reply:
Yes, it's intentional that it's unused.